docs/rfcs/XXX-pull-request-review.md

```diff
@@ -0,0 +1,145 @@
+* Clicking the review summary bubble opens an `IssueishPaneItem` in the workspace center, open to the reviews tab.
+* Clicking a line comment opens or activates an editor on the referenced file and scrolls to center the comment's line, translated according to local changes if appropriate.
+* Line comments within the review are rendered: _with a dot_ before the file has been opened and the corresponding decoration is visible; _with no icon_ after the file and decoration have been seen; and _with a checkmark_ after the comment has been marked "resolved" with the control on its decoration.
+
+If a new review has been started locally, it appears at the top of the "Reviews" section within this tile:
+
+![pending-review](https://user-images.githubusercontent.com/378023/46275946-9bd69680-c599-11e8-9889-66c35458286a.png)
+
+* The review summary is a TextEditor that may be used to compose a summary comment.
+* Choosing "Cancel" dismisses the review and any comments made. If there are local review comments that will be lost, a confirmation prompt is shown first.
+* Choosing "Submit review" submits the drafted review to GitHub.
+
+### Non-current pull request tiles
+
+Pull request tiles other than the current pull request display a one-line review summary, showing the number of accepting, comment, and change-request reviews made on each. Clicking the review summary opens the `IssueishPaneItem` for that pull request and opens the reviews tab.
+
+![non-current pull request tile](https://user-images.githubusercontent.com/17565/46228625-a2aea080-c330-11e8-945b-72be7824623f.png)
+
+### IssueishPaneItem "Changes" tab
+
+Each `IssueishPaneItem` opened on a pull request has a "Changes" tab that shows the full PR diff, annotated with comments from all reviews.
+
+![changes-tab](https://user-images.githubusercontent.com/378023/46287431-6e9bdf80-c5bd-11e8-99eb-f3f81ba64e81.png)
+
+* The up and down arrow buttons quickly scroll to center the next or previous comment within this tab.
+* Clicking the :hamburger: button navigates to the "Reviews" tab, expands the owning review, and scrolls to the same comment within that view.
+* Clicking the "code" (`<>`) button opens the corresponding file in a TextEditor and scrolls to the review comment decoration there.
+* Clicking within the "Reply..." text editor expands the editor to several lines and focuses it.
+* Clicking "mark as resolved" marks the comment as resolved with on GitHub. If the "reply..." editor has non-whitespace content, it is submitted as a final comment first.
+* The "comment" button is disabled unless the "reply" editor is expanded and has non-whitespace content.
+* Clicking "comment" submits the response as a new stand-alone comment on that thread.
+
+Hovering in the diff's gutter reveals a `+` icon that allows users to begin creating a new review with the same UI as described in the "In-editor decorations" section.
+
+### IssueishPaneItem "Reviews" tab
```

Comment:
> The contents of the PR comments could be relevant to review comments.

That's an interesting question. Sometimes I wonder how it would feel if PR and Review comments are strictly separated?

- PR comments: More organizational, things to do, build failing etc.
- Review summary and review comments: Discussion only about the content, the changes.

The only problem, I think you mentioned earlier, there is no place to make review comments without choosing a line. Review summaries could be used, but don't have replies and I think get overridden, when making a new review.

Somehow I'm still intrigued to try the separation, maybe because reading reviews in the conversion AND changes feels like double work?